### DISCUSSION TITLE
Project Geometry onto Sketch

### DISCUSSION THREAD

**Discussion by InsomniacA521** [hlorus/CAD_Sketcher#108] [Feature Requests]
Motivation / Existing Problem
A feature similar to the project geometry tool in AutoCAD Inventor, which allows you to project lines and points from objects outside of the sketch onto your working plane.

Description
A tool that allows you to select lines/points from objects outside of the sketch and project them onto the working plane. Ideally these projected lines/points would be locked in position relative to the object they originated from.

Alternative Solutions
The only alternative solution I can think of is to draw the geometry manually by translating the point coordinates of the object-lines to point coordinates within the sketch. 

Mockup
![projectGeometry](https://user-images.githubusercontent.com/60302704/167186930-216c2c7b-12a7-452a-8dcf-76515f6a7327.jpg)

**Comment by mullcom** [comment]
This is a must have fetuer

**Reply by Arch-druid** [reply]
I think it's coming in the next version
